aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -291,8 +291,15 @@ std::tuple<Tensor, Tensor> max(const Tensor& self, int64_t dim, bool keepdim) {
   if (self.is_quantized()) {
     Tensor max = at::empty({0}, self.options().dtype(toUnderlying(self.scalar_type())));
     at::native::max_out(max, max_indices, self.int_repr(), dim, keepdim);
-    // TODO: qscheme
-    return std::tuple<Tensor, Tensor>(at::_make_per_tensor_quantized_tensor(max, self.q_scale(), self.q_zero_point()), max_indices);
+    // TODO: unify tensor APIs for different qschemes and
```

Comment:
If we are doing max for per-channel quantized tensors, doing the max on the int repr is not correct as each channel dimension is scaled differently.